firmware/components/driver_bus_i2c/driver_i2c.c

```diff
@@ -12,288 +12,287 @@
 
 #ifdef CONFIG_DRIVER_I2C_ENABLE
 
-#define I2C_MASTER_NUM             I2C_NUM_1
-#define I2C_MASTER_TX_BUF_DISABLE  0
-#define I2C_MASTER_RX_BUF_DISABLE  0
+#define I2C_MASTER_NUM I2C_NUM_1
```

Comment:
How is this better? I like the aligned stuff.

Reply:
Then propose a change to the clang-format file as I asked you to ;) This is a starting point and I want to get somewhere where the builds pass.

Reply:
Okay, clang-format 10 has an option for this (AlignConsecutiveMacros) - 7 doesn't yet. I'll figure out if 8 does.

Reply:
It does not - the patch is rather recent. I'd say we should just keep the option in the file for now.

Reply:
The current version has aligned macros, so once that's released that'll work :)